labonneboite/importer/jobs/check_dpae.py

```diff
@@ -1,10 +1,10 @@
 import sys
-import os
 
 from labonneboite.importer import settings
 from labonneboite.importer import util as import_util
 from labonneboite.importer.util import parse_dpae_line
-from .common import logger
+from labonneboite.importer.jobs.common import logger
```

Comment:
y u no like `from .common import logger` ?

Reply:
tried it, kept trying it, could not make it work for some weird reason, sorry dude!

Reply:
je vais retenter, on sait jamais

Reply:
confirmé: `ValueError: Attempted relative import in non-package`  pour `make run_importer_jobs`  une idée docteur?

Reply:
Non, y a pas moyen si on exécute le script avec `python jobs/check_dpae.py`. Pour que ça marche il faudrait ajouter 'check_dpae = labonneboite.importer.jobs.check_dpae:run'` dans le `console_scripts` de `setup.py` et appeler la commande `check_dpae`.